<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControllerSettingsDialog</class>
 <widget class="QDialog" name="ControllerSettingsDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1274</width>
    <height>668</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PCSX2 Controller Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
   <item row="0" column="0">
    <widget class="QListWidget" name="settingsCategory">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Global Settings</string>
      </property>
      <property name="icon">
       <iconset theme="settings-3-line">
        <normaloff>.</normaloff>.</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Controller 1</string>
      </property>
      <property name="icon">
       <iconset theme="gamepad-line">
        <normaloff>.</normaloff>.</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Controller 2</string>
      </property>
      <property name="icon">
       <iconset theme="gamepad-line">
        <normaloff>.</normaloff>.</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hotkeys</string>
      </property>
      <property name="icon">
       <iconset theme="keyboard-line">
        <normaloff>.</normaloff>.</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="settingsContainer">
     <property name="minimumSize">
      <size>
       <width>1100</width>
       <height>620</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="page"/>
     <widget class="QWidget" name="page_3"/>
     <widget class="QWidget" name="page_4"/>
     <widget class="QWidget" name="page_5"/>
     <widget class="QWidget" name="page_6"/>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
